docs/source/extending/parsers.rst

```diff
@@ -87,15 +87,15 @@ To add a new fitting problem definition type, complete the following steps:
      and works in a similar way. 
 
    - **Integration tests**: Add an example to the directory
-     ``fitbenchmarking/mock_problems/all_parser_set/``.
+     ``fitbenchmarking/test_files/all_parser_set/``.
      This will be used to verify that the problem can be run by scipy, and that
      accuracy results do not change unexpectedly in future updates.
      If the software used for the new parser is pip-installable, and the
      installation is done via FitBenchmarking's ``setup.py``, then add the
-     same example to ``fitbenchmarking/mock_problems/default_parsers/``.
+     same example to ``fitbenchmarking/test_files/default_parsers_set/``.
 
      As part of this, the ``systests/expected_results/all_parsers.txt`` file,
-      and if necessary the ``systests/expected_results/default_parsers.txt`` file,
+      and if necessary the ``systests/expected_results/default_parsers_set.txt`` file,
```

Comment:
Are these now csv? @Shanrahan16

Reply:
Yes, sorry, I updated it in the other one I think, so I thought it had picked them all up as conflicts- I've changed it now though, and checked there are no more like this :)